src/mixxx.cpp

```diff
@@ -261,6 +261,7 @@ void MixxxMainWindow::initialize(QApplication* pApp, const CmdlineArgs& args) {
                              m_pRecordingManager);
     m_pPlayerManager->bindToLibrary(m_pLibrary);
 
+    m_pLibrary->scan();
```

Comment:
The feature can be disabled at startup so it does not interfere with the start of a performance. If the missing files that show incorrectly show in Mixxx were easily listed, this bug would be a simple fix. But I don't think there is such a list, unless you figure out a way to create one without the overhead of running the whole library scanner.